src/qt/forms/debugwindow.ui

```diff
@@ -1077,14 +1077,17 @@
                 </widget>
                </item>
                <item row="1" column="0">
-                <widget class="QLabel" name="label_23">
+                <widget class="QLabel" name="peerConnectionTypeLabel">
+                 <property name="toolTip">
+                   <string>The type of peer connection:&lt;ul&gt;&lt;li&gt;Inbound: initiated by peer&lt;/li&gt;&lt;li&gt;Outbound Full Relay: default&lt;/li&gt;&lt;li&gt;Outbound Block Relay: does not relay transactions or addresses&lt;/li&gt;&lt;li&gt;Outbound Manual: added using RPC %1 or %2/%3 configuration options&lt;/li&gt;&lt;li&gt;Outbound Feeler: short-lived, for testing addresses&lt;/li&gt;&lt;li&gt;Outbound Address Fetch: short-lived, for soliciting addresses&lt;/li&gt;&lt;/ul&gt;</string>
```

Comment:
translation nit: How are translators supposed to translate `:&lt;ul&gt;&lt;li&gt;`? It might be better to say "Can be one of %s" and then use `Join(translated_strings, ", ")` or simply "Refer to the getpeerinfo RPC help for details on the connection types"?

Reply:
@hebasto, if we replace the escaped markup with placeholders (as done here for the nonbreaking hyphen), is that still QT Designer-friendly? Can do it in #179.

Reply:
> translation nit: How are translators supposed to translate `:&lt;ul&gt;&lt;li&gt;`?

There was a discussion in private IRC with @jonatack:
> \<hebasto\>
> 14:24:52 but I expect the translators' burden :)
> 14:25:39 hope that translators are aware of escaping :)

I'm not an expert in translation but I believe that `:&lt;ul&gt;&lt;li&gt;` is not a subject to translate as `%1` or `%2`.

> It might be better to say "Can be one of %s" and then use `Join(translated_strings, ", ")` or simply "Refer to the getpeerinfo RPC help for details on the connection types"?

Either of the suggested variants LGTM.

> if we replace the escaped markup with placeholders (as done here for the nonbreaking hyphen), is that still QT Designer-friendly?

I think it is. But, tbh, I'd prefer more succinct @MarcoFalke's suggestions.

Reply:
Thanks. Helpfulness and readability for users were the reasons for the iterations above and it seems best to not regress on those improvements with referring GUI users to the command line or dropping the readable list layout. I'll propose replacing the escaped markup with placeholders.

Reply:
Why not a single placeholder for the whole unordered list? We still have a (shorter) tooltip in the form file, and make translators' life easier :)

Reply:
SGTM too.

Reply:
Done in #180